### ISSUE TITLE
Community Committee Chairperson September 2018 nominations

### ISSUE THREAD

**Issue by hackygolucky** [nodejs/community-committee#363]
### It's election time for the Chairperson of Community Committee.

Please submit your nominations to [this form](https://docs.google.com/forms/d/e/1FAIpQLSdqKLnXmj3XWWO0UpChZS-zmZDrUfml52Jn95FSKyWjRWjWLQ/viewform?usp=sf_link) by *Monday. August 27 at 11:59pm PT*. Nominations will be published after the date passes so that candidates can be considered for election.  Please do not publicly share you are running prior to this date.

> The CC will elect from amongst voting CC members a CC Chairperson to work on building an agenda for CC meetings and collaborate with the Individual Membership Directors the wishes of the CC to the Board for a term of one year according to the Node.js Foundation’s By-laws. The CC shall hold annual elections to select a CC Chairperson; there are no limits on the number of terms a CC Chairperson may serve.

## Nominated candidates:
- your name here @-handle

As stated in the previous election, expected responsibilities include but are not limited to:
- Create each meeting's agenda. 
- Create the event and make sure it is live-streamed in YouTube. 
- Chair and facilitate each meeting. It is the chairperson's responsibility to ensure that the meeting stays on track, sticks to the agenda, and finishes within the allotted time.
- Collaborate with the Individual Membership Directors(currently @williamkapke and @ashleygwilliams) on presenting requests to the Board from CommComm that need attention. 
- Collaborate with the TSC Chair and Director
- Coordinate efforts in shared repos, as listed in Admin

All Community Committee active members are eligible to self-nominate. If you are having trouble with voting, please email tracy@nodejs.org.

**Comment by Tiriel** [comment]
Quick nit: Maybe it should be specified that the nominations are self-nominations?
It should be pretty obvious, but for a time I was sure and wondered if I could nominate someone else.

**Comment by hackygolucky** [comment]
@Tiriel Yeah, I'll specify. I like to think it's not a great thing to nominate someone else to do a lot of work :P (even if you've talked to them about it!)

**Comment by Tiriel** [comment]
Indeed a poor joke :smile:

**Comment by bnb** [comment]
I'd like to note that I don't intend on re-running for the Community Committee chairperson role again.

From when I ran the first time, my hope was that I would be able to help kick-start the Community Committee into a group that was self-driving and self-motivated. I've had this in the back of my mind in every interaction, every discussion, and every meeting for the past few months knowing that we were fast approaching the election.

Having worked with all of you – the Community Committee members – having seen everyone grow and take on awesome things that I didn't think we'd even come close to approaching while I was chair, I'd absolutely say that we have achieved that.

I'd like to make room for someone else to help drive the administrative side of the Community Committee along with @hackygolucky. In my opinion, having natural leadership turnover in this kind of role is both healthy for the leadership and healthy for the broader group. There are plenty of new things that we can and will do – and fresh leadership is always a positive, especially for a group so new (we really are still very new!)

For context, I'd like to share some of the things that _I_ have done almost every week, and give y'all a rough estimate of the time commitment:

- 5-6 hours of async/sync communication with CommComm members (Twitter DM, Slack DM) – communication and transparency is probably one of the most important parts of this role. 
- ~3-4 hours every week for various Initiative meetings and the top level CommComm meeting
- ~4-5 hours reviewing issues, PRs, Node.js emails, and other communication media 
- 1 hour every other week for the Board Subcommittee meeting
- Roughly 2 hours every week for one-off meetings that aren't really planned but do indeed take up time when there are fires

In total, that's about 15-18 hours a week that I spend on CommComm work. I'm also not including travel in this because I don't know if I can accurately represent the amount of needed travel for Node.js focused events since I am currently enabled to do a lot of it through my role at NodeSource. That said, it is a vital part of the role in my opinion.

It's worth noting that I would **highly recommend** being able to work on this as a part of your day job. I've set the precedent of open DMs – this is generally useful because we have people around the globe. While you don't need to follow this precedent, I am not sure of how it will or will not affect other parts of your work in the role.

I am exceedingly excited to see who the next Chairperson is, and will make myself available to them in the hopes that I can help out with guidance/advice/mentorship as much as possible :heart:

**Comment by joesepi** [comment]
What are next steps here? Thanks.

**Comment by bnb** [comment]
Per the form:

> Nominations will be announced Monday, August 27.

I believe this will occur once @hackygolucky is back 👍
